Packs/GroupIB_ThreatIntelligenceAttribution/IncidentFields/incidentfield-GIB_Bulletin_Family.json

```diff
@@ -0,0 +1,59 @@
+{
+	"aliasTo": "",
+	"aliases": null,
+	"associatedToAll": false,
+	"associatedTypes": [
+		"GIB OSI Vulnerability"
```

Comment:
Is this used only in GIB OSI Vulnerability incidents? Can / should it be used in "GIB Cybercriminal Threat" and "GIB Nation-State Cybercriminals Threat" incidents?

Reply:
We have tested the solution several times with our team and think it is better to leave it only to the GIB OSI Vulnerability